spec/services/mhv_account_type_service_spec.rb

```diff
@@ -5,12 +5,14 @@
 RSpec.describe MHVAccountTypeService do
   subject { described_class.new(user) }
 
+  let(:unknown_error) { 'BackendServiceException: {:status=>400, :detail=>nil, :code=>"VA900", :source=>nil}' }
```

Comment:
This is the best we can do since MHV doesn't provide any error payload whatsoever, just a bare status, in some conditions.